tests/zfs-tests/include/libtest.shlib

```diff
@@ -4066,3 +4066,17 @@ function get_arcstat # stat
 		return 1
 	fi
 }
+
+function get_iostat # pool stat
+{
+	if is_linux; then
+		typeset pool=$1
+		typeset stat=$2
+		typeset zfs_iostats="/proc/spl/kstat/zfs/${pool}/iostats"
+		[[ -f "$zfs_iostats" ]] || return 1
+		grep $stat $zfs_iostats | awk '{print $3}'
+		return $?
+	else
+		return 1
+	fi
+}
```

Comment:
You can use the `get_trim_io` and `verify_trim_io` functions in ` trim.kshlib` to get these functions in a portable way.  You could even extend them to target just the TRIM status for the L2ARC device by including it in the `zpool iostat` command.  i.e. `zpool iostat -pr <pool> <l2arc vdev>`.